addon/helpers/format-date.js

```diff
@@ -0,0 +1,19 @@
+import { helper } from '@ember/component/helper';
+import dayjs from 'dayjs';
+
+export function formatDate([date, format], { type }) {
+  if (!date) {
+    return '';
+  }
+  let outputFormat = 'L LT';
```

Comment:
The format we've used most of the time is `l LT` - this drops the leading 0s from days and months.
```suggestion
  let outputFormat = 'l LT';
```

Reply:
We got together and discussed it, and are going to use `L LT` as our new standard.